code/game/gamemodes/handofgod/_handofgod.dm

```diff
@@ -193,8 +202,9 @@ var/global/list/global_handofgod_structuretypes = list()
 	if(colour == "blue")
 		blue_deity_followers += follower_mind
 
-	H.faction |= "[colour] god"
+	H.faction += "[colour] god"
```

Comment:
I assumed it did |= so you could swap between the two different colored teams, wouldn't this mean you could potentially become both?